### ISSUE TITLE
uncompatible with react 17

### ISSUE THREAD

**Issue by spirituelle** [bl00mber/react-phone-input-2#384]
while i was trying to install the module i get the uncompatible verssion with the newest react 17 version with the internal 16 version.
need a full update then or we go back to react 16 😩

**Comment by RSRajendra** [comment]
try
npm install -f

it worked for me

**Comment by RSRajendra** [comment]
Also remove yarn.lock or package-lock.json befor running. npm install -f

**Comment by spirituelle** [comment]
I removed package-lock.json and run npm install -f but it still returning same error (ARE YOU SURE YOUR PROJECT IS UNDER REACT 17)
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: static-landing@0.1.0
npm ERR! Found: react@17.0.1
npm ERR! node_modules/react
npm ERR!   react@"17.0.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.12.0" from react-phone-input-2@2.13.9
npm ERR! node_modules/react-phone-input-2
npm ERR!   react-phone-input-2@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /.npm/_logs/2021-02-15T21_55_44_287Z-debug.log
